pkg/roachprod/install/cockroach.go

```diff
@@ -1294,9 +1295,13 @@ func (c *SyncedCluster) setClusterSettings(
 func (c *SyncedCluster) generateClusterSettingCmd(
 	ctx context.Context, l *logger.Logger, node Node, virtualCluster string,
 ) (string, error) {
-	if config.CockroachDevLicense == "" {
-		l.Printf("%s: COCKROACH_DEV_LICENSE unset: enterprise features will be unavailable",
-			c.Name)
+	license := config.CockroachDevLicense
+	if license == "" {
+		license, _ = (&licenseccl.License{
+			Type:              licenseccl.License_Enterprise,
+			Environment:       licenseccl.Development,
+			ValidUntilUnixSec: timeutil.Now().AddDate(0, 1, 0).Unix(),
```

Comment:
Is there any reason not to make this a perpetual license (e.g., `AddDate(1, 0, 0)`)? I'm thinking of some edge-cases, e.g., a long-running cluster.

Reply:
We can, but since it’s an enterprise license, throttling does not apply. Once it expires, there are no product restrictions, aside from the DB console indicating that the license has expired.